spec.html

```diff
@@ -7536,6 +7535,18 @@ <h1>SetFunctionName ( _F_, _name_ [ , _prefix_ ] )</h1>
     </emu-clause>
 
     <!-- es6num="9.2.12" -->
+    <emu-clause id="sec-setfunctionlength" aoid="SetFunctionLength">
+      <h1>SetFunctionLength ( _F_, _length_)</h1>
+      <p>The abstract operation SetFunctionLength requires a Function argument _F_, and a Number argument _length_. This operation adds a `length` property to _F_ by performing the following steps:</p>
+      <emu-alg>
+        1. Assert: _F_ is an extensible object that does not have a `length` own property.
+        1. Assert: Type(_length_) is Number.
+        1. Assert: ToLength(_length_) is equal to _length_.
+        1. Return ! DefinePropertyOrThrow(_F_, `"length"`, PropertyDescriptor{[[Value]]: _length_, [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true*}).
+      </emu-alg>
+    </emu-clause>
+
+    <!-- es6num="9.2.13" -->
```

Comment:
To my understanding we should not be moving/adding es6num since it is referring to where a clause was in the ES2015 specification: http://www.ecma-international.org/ecma-262/6.0/